src/runtime/node/cluster/index.ts

```diff
@@ -0,0 +1,82 @@
+import { notImplemented } from "src/runtime/_internal/utils";
+import noop from "../../mock/noop";
+import mock from "../../mock/proxy";
+import type cluster from "node:cluster";
+import type { Cluster, Worker as _Worker } from "node:cluster";
+import { EventEmitter } from "../events";
+
+const _cluster = new EventEmitter() as Cluster;
+const _noop = () => {
+  return _cluster;
+};
+
+const on: typeof cluster.on = _noop;
+const addListener: typeof cluster.addListener = _noop;
+const once: typeof cluster.once = _noop;
+const removeListener: typeof cluster.removeListener = _noop;
+const removeAllListeners: typeof cluster.removeAllListeners = _noop;
+const emit: typeof cluster.emit = () => false;
+const off: typeof cluster.off = _noop;
+const prependListener: typeof cluster.prependListener = _noop;
+const prependOnceListener: typeof cluster.prependOnceListener = _noop;
+const listeners: typeof cluster.listeners = function (name) {
+  return [];
+};
+const listenerCount: typeof cluster.listenerCount = () => 0;
+const setMaxListeners: typeof cluster.setMaxListeners = notImplemented(
```

Comment:
why are certain things noops and others throw? wouldn't throwing be considered a breaking change compared to the current auto-mocking behavior?